packages/commonwealth/client/scripts/state.ts

```diff
@@ -230,95 +230,93 @@ export async function initAppState(
   updateSelectedChain = true,
   customDomain = null
 ): Promise<void> {
-  return new Promise((resolve, reject) => {
-    $.get(`${app.serverUrl()}/status`)
-      .then(async (data) => {
-        app.config.chains.clear();
-        app.config.nodes.clear();
-        app.user.notifications.clear();
-        app.user.notifications.clearSubscriptions();
-
-        data.result.nodes
-          .sort((a, b) => a.id - b.id)
-          .map((node) => {
-            return app.config.nodes.add(NodeInfo.fromJSON(node));
-          });
-
-        data.result.chainsWithSnapshots
-          .filter((chainsWithSnapshots) => chainsWithSnapshots.chain.active)
-          .map((chainsWithSnapshots) => {
-            delete chainsWithSnapshots.chain.ChainNode;
-            return app.config.chains.add(
-              ChainInfo.fromJSON({
-                ChainNode: app.config.nodes.getById(
-                  chainsWithSnapshots.chain.chain_node_id
-                ),
-                snapshot: chainsWithSnapshots.snapshot,
-                ...chainsWithSnapshots.chain,
-              })
-            );
-          });
-
-        app.roles.setRoles(data.result.roles);
-        app.config.notificationCategories =
-          data.result.notificationCategories.map((json) =>
-            NotificationCategory.fromJSON(json)
-          );
-        app.config.chainCategories = data.result.chainCategories;
-        app.config.chainCategoryTypes = data.result.chainCategoryTypes;
+  axios
+    .get(`${app.serverUrl()}/status`)
+    .then(async ({ data }) => {
```

Comment:
This is a change. Most of the rest of the diff is due to an indentation change, from tweaking the promise format. I will try to highlight any actual changes.